### PR TITLE
chore: update zone.js version range package.json

### PR DESCRIPTION
**Pull request by ptu14**
I need this in `6.5.17`

## What I did

I updated the zone.js version scope in package.json to `"^0.8.29 || >=0.9.0 <1.0.0"` to enable the use of Angular 15 and NX version 15. Angular 15 requires `zone.js` version `0.12.0`, which I could not use without this update.

I did the same as: https://github.com/storybookjs/storybook/commit/75e57ad1deacdf5a6cf9c49b05bed591685e0b75

## Checklist

- [x] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [x] Make sure to add/update documentation regarding your changes
- [x] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

can be labelled `dependencies`

### CODE REVIEW

**Comment by ptu14** [comment]
In issue #21737, I noticed that I'm not the only one using `zone.js` `0.12.0` with Storybook `6.5.16`. In my case, updating to Storybook `7.0` won't be possible for a few more months.

**Comment by skalawi** [comment]
Is there any chance to release it? I have the same problem

**Comment by skorzyk** [comment]
The same problem

File: app/angular/package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
     "@nrwl/workspace": ">=11.1.0",
     "rxjs": "^6.0.0 || ^7.4.0",
     "typescript": "^3.4.0 || >=4.0.0",
-    "zone.js": "^0.8.29 || ^0.9.0 || ^0.10.0 || ^0.11.0"
+    "zone.js": "^0.8.29 || >= 0.9.0 < 1.0.0"
```
Comment (ptu14) [inline app/angular/package.json:123]: `^0.8.29 || >= 0.9.0 < 1.0.0` can be replaced by `>=0.8.29 < 1.0.0`, but wants to be consistent with Storybook version 7

**Review by valentinpalkovic — APPROVED** [review]

**Comment by valentinpalkovic** [comment]
Hey @ptu14,

Is the workaround mentioned in https://github.com/storybookjs/storybook/issues/20758#issuecomment-1411820252 working for you? If so, please see my answer here https://github.com/storybookjs/storybook/issues/20758#issuecomment-1532832277 related to this topic.

**Comment by ptu14** [comment]
OK thanks. It's not a perfect solution for me, but I understand your approach. So I will have to push my team to update the nx version. Fell free to close this PR

**Comment by valentinpalkovic** [comment]
@ptu14 Thank you for your understanding

**Pull request closed without merging**